target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -78,6 +78,10 @@ netgear,r6800)
 	ucidef_set_led_netdev "lan3" "LAN3" "white:lan3" "lan3"
 	ucidef_set_led_netdev "lan4" "LAN4" "white:lan4" "lan4"
 	;;
+raisecom,msg1500-x-00)
+	ucidef_set_led_netdev "wlan2g" "Wi-Fi 2.4G" "blue:wlan2g" "wlan0" "link"
+	ucidef_set_led_netdev "wlan5g" "Wi-Fi 5G" "blue:wlan5g" "wlan1" "link"
```

Comment:
_DTS_ trigger, e.g. https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/dts/mt7621_tplink_rexx0-v1.dtsi#L66

This should replace the entries in 01_leds entirely, so they can be dropped.

Reply:
Thanks for your advice. "phyxradio" is appropriate after testing.